src/shared/components/Gallery/Gallery.style.ts

```diff
@@ -31,3 +34,24 @@ export const CarouselArrowsContainer = styled.div`
   grid-column-gap: ${sizes(4)};
   margin-left: auto;
 `
+
+export const TitleWrapper = styled.div`
+  align-items: baseline;
+  display: flex;
+`
+
+export const SeeAllLink = styled(Button)`
+  margin-left: ${sizes(9)};
+
+  :hover {
+    text-decoration: none;
+  }
+`
+
+export const StyledSvgGlyphPlay = styled(SvgPlayerPlay)`
+  margin-right: ${sizes(3)};
```

Comment:
```suggestion
```
I think the space between icon and text is too broad. The default button margin should be fine in this case